homeassistant/components/rabbitair/manifest.json

```diff
@@ -0,0 +1,21 @@
+{
+  "domain": "rabbitair",
+  "name": "Rabbit Air",
+  "config_flow": true,
+  "documentation": "https://www.home-assistant.io/integrations/rabbitair",
+  "requirements": [
+    "python-rabbitair==0.0.6"
+  ],
+  "ssdp": [],
+  "zeroconf": [
+    "_rabbitair._udp.local."
+  ],
+  "homekit": {},
+  "dependencies": [
+    "zeroconf"
+  ],
```

Comment:
Not sure about this. The [docs](https://developers.home-assistant.io/docs/network_discovery/#mdnszeroconf) say that we should add `zeroconf` to `dependencies` if we use `async_get_async_instance`. Perhaps with the `zeroconf` section, this dependency is added implicitly, but I have not found this documented in any way. However, I have noticed that most integrations specify it in `after_dependencies` (i.e. as an optional dependency), and this is more appropriate since having mDNS functionality is not an absolute requirement.